src/main/java/digital/slovensko/autogram/core/errors/BatchConflictException.java

```diff
@@ -0,0 +1,7 @@
+package digital.slovensko.autogram.core.errors;
+
+public class BatchConflictException extends AutogramException {
+    public BatchConflictException(String message) {
+        super("Iné hromadné podpisovanie už prebieha", "", message);
```

Comment:
Tieto errory už začínajú byť tricky, keďže sa ich snažíme použiť aj pre GUI aj pre API response, kde je to v SK a v EN. To asi doriešime v ďalšom tasku ohľadom internacionalizácie. Inak, naschvál je vynechaný subheading vo všetkých týchto nových erroroch? Práve subheading sa tiež posielal v response.

Reply:
Po pravde neviem vymysliet tie subheadingy :D lebo je to tak ze by som tam napisal 3 krat to iste. Rad prijmem navrhy

Reply:
Všeobecne by to mal byť trochu rozšírený heading alebo jeho krátke dovysvetlenie. Ale prežijeme aj aspoň s duplikátom headingu, ak to inak nejde :D Napr. ut by som dal `Naraz je možné vykonávať iba jedno hromadné podpisovanie`.